src/main/java/frc/robot/Robot.java

```diff
@@ -104,8 +105,6 @@ public void robotInit() {
 
         Shuffleboard.getTab("Shuffleboard").add("Autonomous", autoChooser);
 
-        DashboardUtils.logTelemetry();
```

Comment:
This is also only fixed if you inherit from `CommandRobot` and on line 84 add `super.robotInit();`